archive/aws/client/theme/colors.js

```diff
@@ -13,6 +13,7 @@ export default {
   textSecondaryDark: '#A8A8A8',
   textSecondaryLight: '#ECECEC',
   textFocus: '#87C8BC',
+  formInput: '#29404F1A',
```

Comment:
this is in the wrong file: `archive/aws/client/theme/colors.js`

but also @edwinjue and @ryanfchase, i was wondering why this folder exists in the first place? why not move it to a separate branch or remove it entirely? in my experience it tends to confuse the vscode autocomplete / language server

Reply:
Good catch.  I went ahead and updated it.

Reply:
@jaasonw @bphan002 just made this ticket, will this address the issue? https://github.com/hackforla/311-data/issues/1690